packages/functions/src/testUtils/TestSamples.ts

```diff
@@ -12,10 +12,11 @@ export const sampleUser = {
   testUser: false,
 };
 
-export function saveTestDonor(donorId: string) {
+export function saveTestDonor(donorId: string, donorFields?: Partial<DbDonor>) {
```

Comment:
This already saves first and last names, why not use them?

Reply:
I want to expect() a test-specific variable

Reply:
Umm... ok.

You can just expect the existing first and last name, but I don't think this is critical